src/common/maps/instruction.element.generic.type.map.ts

```diff
@@ -0,0 +1,69 @@
+import { InstructionElementGenericType } from '../enum/instruction.element.generic.type'
+
+export class InstructionElementGenericTypeMap {
+  predefined_program_options = {
+    name: 'Predefined Program Options',
+    options: [
+      {
+        id: null,
+        name: InstructionElementGenericType.SYSTEM,
+        type: InstructionElementGenericType.SYSTEM,
+      },
+      {
+        id: null,
+        name: InstructionElementGenericType.ASSOCIATED_TOKEN,
+        type: InstructionElementGenericType.ASSOCIATED_TOKEN,
+      },
+      {
+        id: null,
+        name: InstructionElementGenericType.TOKEN,
+        type: InstructionElementGenericType.TOKEN,
+      },
+    ],
+  }
+  predefined_account_options = {
```

Comment:
Naming does not follow project rules

`ESLint: Class Property name `predefined_sysvar_options` must match one of the following formats: camelCase, UPPER_CASE, PascalCase(@typescript-eslint/naming-convention)`